### PR TITLE
Add dynamic og:image and remove canonical urls

### PR DESCRIPTION
**Pull request by tifa2UP**
### Summary 
1. Removed the canonical url property `og:url` which made Facebook and other crawlers always fetch the meta tags for https://quran.com instead of the desired path.

2. Added dynamic open-graph images that fetch image previews based on the content.

### Test Plan
@mmahalwy I don't have the API running locally so haven't been able to test the changes. Let's get it up on staging and give it a test before we deploy it to prod.

### CODE REVIEW

File: src/config.js [hunk 1]
```diff
@@ -115,17 +115,13 @@ const config = {
           property: 'og:description',
           content: description
         },
-        {
-          property: 'og:url',
```
Comment (tifa2UP) [inline src/config.js:119]: @mmahalwy this is what was causing the canonical URL generation

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32795

File: src/containers/ChapterInfo/index.js [hunk 2]
```diff
@@ -23,6 +23,7 @@ const ChapterInfo = ({ chapter, info }) => (
     <Helmet
       {...makeHeadTags({
         title: `Surah ${chapter.nameSimple} [${chapter.chapterNumber}]`,
+        image: `https://og.tarteel.io/${chapter.chapterNumber}`,
```
Comment (tifa2UP) [inline src/containers/ChapterInfo/index.js:26]: it's an alias for the vercel deployment

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32796

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32797

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32798

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32799

File: src/containers/ChapterInfo/index.js [hunk 3]
```diff
@@ -23,6 +23,7 @@ const ChapterInfo = ({ chapter, info }) => (
     <Helmet
       {...makeHeadTags({
         title: `Surah ${chapter.nameSimple} [${chapter.chapterNumber}]`,
+        image: `https://quran-og-image.vercel.app/${chapter.chapterNumber}`,
```
Comment (mmahalwy) [inline src/containers/ChapterInfo/index.js:26]: haha nice :)

**Pull request merged**